index.js

```diff
@@ -31,7 +33,13 @@ function dnxRunner(dnxCommand, options) {
   }
 
   if (options.run === true) {
-    commands.push('@powershell -NoProfile -ExecutionPolicy unrestricted -Command "for(;;) { Write-Output \"Starting...\"; dnx --watch . ' + dnxCommand + ' }"');
+    var xcmd = 'dnx --watch . ' + dnxCommand;
+    console.log('xcmd', xcmd);
+    if (isWin) {
+      commands.push('@powershell -NoProfile -ExecutionPolicy unrestricted -Command "for(;;) { Write-Output \"Starting...\"; ' + xcmd + ' }"');      
+    } else {
+      commands.push(xcmd);
```

Comment:
Hmm, this will not restart the server I assume as you only run it once. Can we put this inside an infinite for loop as we do in powershell case?

Also, I am not sure if `--watch` works outside the Linux now.